docs/add_node.md

```diff
@@ -0,0 +1,70 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Add additional nodes](#add-additional-nodes)
+  - [How to add a new node after cluster creation completed](#how-to-add-a-new-node-after-cluster-creation-completed)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Add additional nodes
+## How to add a new node after cluster creation completed
+after cluster creation with command `clusterctl`, machines can be queried by:
+```
+# kubectl --kubeconfig=kubeconfig  get machines
+NAME                  PROVIDERID              PHASE
+jichen-master-g2x5d   xxxxxxxx                Running
+jichen-node-s9s2b     xxxxxxxx                Running
+```
+
+In order to add additional node, need create a machine definition,
+refer to `examples/ibmcloud/out/machines.yaml` for more info.
+
+Here's a sample and we can put into `~/temp.yaml`
```

Comment:
What about an example file under examples or use here document instead?

Reply:
I prefer to use document first... all other providers seems not provide this in template.